### DISCUSSION TITLE
User and password unknown after upgrade to Windows 11

### DISCUSSION THREAD

**Discussion by Bricolas** [AveYo/MediaCreationTool.bat#310] [Q&A]
I used Windows 1O without password.
I have upgraded from Win 10 to Win 11 using the MediaCreationTool.bat
After the upgrade was completed, the login screen asked for a user and a password.
The screen shows "Other User" with 2 fields : "User" and "Password".
What can I try to login ?

**Comment by AveYo** [comment]
A proper upgrade would get you back to the desktop - something happened.
See #237 last comment by Pldm24
See #154 last comment by joey00797
After you create a new user in safe mode you should be able to login.

But if your 10 account has not been migrated at all, your apps and profile files would be missing
so it's best to get back to windows 10 (Settings app - Update - Windows Update - Recovery)
